tests/pytests/test.py

```diff
@@ -554,22 +554,22 @@ def testExplain(env):
     env.assertEqual(expected, res)
 
     q = '(hello world) "what what" hello|world @bar:[10 100]|@bar:[200 300]'
-    res = r.execute_command('ft.explain', 'idx', q)
+    res = r.execute_command('ft.explain', 'idx', q, 'DIALECT', 2)
```

Comment:
Look at the comment I added to the PR, in my opinion all tests should run with both dialects.